deps/objconv.version

```diff
@@ -0,0 +1,7 @@
+## jll artifact
+# Objconv (we don't ship this, so no need for a fake JLL; therefore we specify the JLL_VER here)
```

Comment:
```suggestion
## jll artifact
# Objconv (we don't ship this, so no need for a fake JLL; therefore we specify the JLL_VER here instead of in a `stdlib/Objconv_jll/Project.toml` file)
```